net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit gnome2-utils unpacker xdg-utils pax-utils
+
+DESCRIPTION="Signal Desktop"
+HOMEPAGE="https://signal.org/ https://github.com/WhisperSystems/Signal-Desktop"
+SRC_URI="https://updates.signal.org/desktop/apt/pool/main/s/${MY_PN}/${MY_PN}_${PV}_amd64.deb"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="ayatana"
+
+RESTRICT="bindist mirror"
+
+RDEPEND="
+	gnome-base/gconf:2
+	dev-libs/nss
+	x11-libs/libXtst
+	net-print/cups
+	ayatana? ( dev-libs/libappindicator:3 )
+	"
+
+QA_PREBUILT="opt/Signal/signal-desktop
+	opt/Signal/libnode.so
+	opt/Signal/libffmpeg.so"
+
+S="${WORKDIR}"
+
+src_install() {
+	doins -r .
```

Comment:
But you didn't set any directory with `insinto`!

Reply:
Done